mgtool/exec.go

```diff
@@ -19,3 +22,24 @@ func Command(_ context.Context, path string, args ...string) *exec.Cmd {
 	cmd.Stdout = os.Stdout
 	return cmd
 }
+
+// Output runs the given command, and returns all output from stdout in a neatly, trimmed manner,
+// or an error if the command failed.
+func Output(cmd *exec.Cmd) (string, error) {
```

Comment:
Let's call it GetOutput?

Reply:
I simply followed the naming used by the command struct :)

Reply:
My bad! :speak_no_evil: :smile:

Reply:
Function is now removed :)